Sources/BuildSystemIntegration/SwiftPMBuildSystem.swift

```diff
@@ -291,6 +293,19 @@ package actor SwiftPMBuildSystem: BuiltInBuildSystem {
       prepareForIndexing: options.backgroundPreparationModeOrDefault.toSwiftPMPreparation
     )
 
+    let pluginScriptRunner = DefaultPluginScriptRunner(
+      fileSystem: localFileSystem,
+      cacheDir: location.pluginWorkingDirectory.appending("cache"),
+      toolchain: hostSwiftPMToolchain,
+      extraPluginSwiftCFlags: [],
```

Comment:
Should we expose this as an option in SourceKitLSPOptions? Otherwise uses who pass `Xbuild-tools-swiftc` on the command line won’t be able to use SourceKit-LSP properly.

Reply:
Yeah, I should have marked this as draft. Need to do that + traits + only change this when background indexing is enabled.